website/content/en/preview/getting-started/getting-started-with-terraform/_index.md

```diff
@@ -42,19 +42,25 @@ After setting up the tools, set the following environment variables to store
 commonly used values.
 
 ```bash
-export CLUSTER_NAME="${USER}-karpenter-demo"
+export CLUSTER_NAME="karpenter-demo"
```

Comment:
Is there a reason to change the cluster name?  Is it just so that we don't have to supply the cluster_name var as an argument to the `terraform apply` command?

Reply:
correct - its not gernally a common way of interacting with Terraform in normal day-to-day use (i.e. - in CI/CD processes) and so I was trying to mirror this practice. 

However, if do want to keep it like this, I would suggest a small tweak to where we re-name the variable to `TF_VAR_cluster_name`. [Terraform will recognize this](https://www.terraform.io/cli/config/environment-variables#tf_var_name) and supply the value for `var.cluster_name` in lieu of supplying `-var="cluster_name=$CLUSTER_NAME"` when planning and applying